### ISSUE TITLE
is_distributed not functioning properly

### ISSUE THREAD

**Issue by coquelin77** [helmholtz-analytics/heat#136]
example (2 processes):
```python
x = ht.ones([2, 2, 2], split=None)
>>> tensor([[1., 1.],
            [1., 1.]])
>>> tensor([[1., 1.],
            [1., 1.]])
x.comm.is_distributed()
>>> is_distributed: True
>>> is_distributed: True
```
Proposed solution: change function to get the tensor split instead of the world size (also move from communication to tensor

print('is_distributed:', x.comm.is_distributed())

**Comment by Markus-Goetz** [comment]
Thanks for the issue. Fixed this already as part of issue #9 I am currently working on. Will take on this ticket.

**Comment by coquelin77** [comment]
okay. i will remove myself then
